src/basho_stats_utils.erl

```diff
@@ -61,9 +61,9 @@ r_port() ->
     try port_command(Port, "write('', file=stdout())\n") of
         _ ->
             receive
-                {Port, {data, {eol, []}}} ->
```

Comment:
I'm also surprised that we don't just end up getting a different dialyzer warning after this change, since the `Port` variable above is still being exported outside the case clause and used on line 65. I tried running `make dialyzer` on a fresh checkout of this repo, but got nothing. @kesslerm would you happen to have the command handy for dialyzing this?